python/examples/ingestion_with_python_config/telemetry_config.py

```diff
@@ -17,20 +18,14 @@
 RULE_NAMESPACES_DIR = Path().joinpath("rule_modules")
 
 
-def nostromos_lv_426() -> TelemetryConfig:
+def nostromos_lv_426() -> Tuple[TelemetryConfig, List[RuleConfig]]:
     named_expressions = load_named_expression_modules(
```

Comment:
Can we get rid of named expressions from the examples? With asset agnostic rules we can start moving away from this